.devcontainer/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  app:
+    extends:
+      file: ./local-test/docker-compose.yaml
```

Comment:
Please rename `.devcontainer/local` directory to mirror the path above (i.e. `.devcontainer/local-test`).

Reply:
Done!